doc/user/content/installation/operational-guidelines.md

```diff
@@ -36,55 +45,16 @@ when operating on datasets larger than main memory as well as allows for a more
 graceful degradation rather than OOMing. Network-attached storage (like EBS
 volumes) can significantly degrade performance and is not supported.
 
-*Additional documentation to come*
```

Comment:
Instead of cluttering up this page, now ported over to cloud provider specific pages.

Later on, will remove this page and just have the cloud specific pages.  But, didn't want to expand the scope too much in this pr.